### PR TITLE
Added BugBug to testing tools

### PR DESCRIPTION
**Pull request by pbylina**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -523,6 +523,7 @@ Table of Contents
   * [Applitools.com](https://applitools.com/) — Smart visual validation for web, native mobile and desktop apps. Integrates with almost all automation solutions (like Selenium and Karma) and remote runners (Sauce Labs, Browser Stack). free for open source. A free tier for a single user with limited checkpoints per week.
   * [Appetize](https://appetize.io) — Test your Android & iOS apps on this Cloud Based Android Phone/Tablets emulators and iPhone/iPad simulators directly in your browser. Free tier includes 1 concurrent session with 100 minutes usage per month. No limit on app size.
   * [Bird Eats Bug](https://www.birdeatsbug.com/) — Report bugs faster (and better). Record your screen with Bird browser extension, it will auto-capture technical data that engineers need to debug. Free tier suitable for small teams.
+  * [BugBug](https://bugbug.io/) - Lightweight test automation tool for web applications. Easy to learn and doesn't require coding. It's free, with unlimited tests. For an additional monthly fee, you also get cloud monitoring and CI/CD integration.
```
Comment (ripienaar) [inline README.md:526]: Please say `It's free, with unlimited tests, for 1 project` or similar wording

thanks!

Comment (pbylina) [reply README.md:526]: We changed our free package, and it's free without project limitations. So it's free in general, but if you want to execute tests in the cloud, you have to pay.

Comment (ripienaar) [reply README.md:526]: OK, then please add that - the point is to add some details about what is the restriction for free accounts.

Comment (pbylina) [reply README.md]: I changed the sentence a bit to be more clear.

**Pull request merged**